addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/GlobalCacheBindingConstants.java

```diff
@@ -37,11 +37,12 @@
     public final static ThingTypeUID THING_TYPE_ITACH_FLEX = new ThingTypeUID(BINDING_ID, "itachFlex");
     public final static ThingTypeUID THING_TYPE_GC_100_06 = new ThingTypeUID(BINDING_ID, "gc100_06");
     public final static ThingTypeUID THING_TYPE_GC_100_12 = new ThingTypeUID(BINDING_ID, "gc100_12");
+    public final static ThingTypeUID THING_TYPE_ZMOTE = new ThingTypeUID(BINDING_ID, "zmote");
```

Comment:
Can you rebase on the current master, fix the merge conflicts and make the order of modifiers for these new lines also `public static final`